### ISSUE TITLE
How can I start this program

### ISSUE THREAD

**Issue by DaZEROrk** [dyedquartz/mangadex-scraper#3]
good day
I need some help and hints on how to start downloading from M-D.org using your program
can you make a wiki or how-To
best regard.

**Comment by dyedquartz** [comment]
you run the program using `mangadex-scraper <ID> <manga/chapter>`. 

so to download https://mangadex.org/title/17176/mahou-shoujo-tokushuusen-asuka

you would enter in `mangadex-scraper 17176 manga`

**Comment by patris-dev** [comment]
For those who don't really understand coding and such, here's some simple steps to get this running:
1. This program runs on Rust, so install that: https://www.rust-lang.org/tools/install
2. You'll also need Visual C++ Tools: https://tinyurl.com/y43zocn5
3. open Command Prompt, type 'cargo install mangadex-scraper' (without quotes)
4. once that all compiles, (still in Command Prompt) you can start using the commands. Use 'cd' to change directory (e.g. 'cd Desktop') - this is where your downloaded files will appear.

**Comment by JoePalffy** [comment]
i don't know, that sounds complicated, if you really wanted to share this with people wouldn't an .exe be the obvious choice ?
no offense but the developer seems to have no idea how bewildering using this program is to the average person.

**Comment by dyedquartz** [comment]
False. there is an exe, you don't need to install rust at all, you run the executable in command prompt

**Comment by JoePalffy** [comment]
ok, so where is this exe ? cuz there is no exe in the files i downloaded from github

**Comment by dyedquartz** [comment]
@JoePalffy my bad, forgot to upload the windows executable, I'll get it up soon

**Comment by JoePalffy** [comment]
Hi, it's been 10 days, any chance you can get around to uploading that exe ?

**Comment by dyedquartz** [comment]
@JoePalffy turns out a library doesn't support windows, I'll see about modifying it so it doesn't require that library

**Comment by JoePalffy** [comment]
ok .... so when you said there is an exe ... was that never tested on the OS it was made for ?

**Comment by dyedquartz** [comment]
@JoePalffy #5
